changelog.md

```diff
@@ -5,6 +5,17 @@
 **Added:**
 - New slider component, see http://backpack.prod.aws.skyscnr.com/components/web/sliders
 
+**Changed:**
+- bpk-component-modal:
+    - Renamed `bpk-modal-scrim` to `bpk-scrim`, and `BpkModalScrim` to `BpkScrim`
```

Comment:
Typically we wouldn't list internal changes like this in the changelog - only public API changes or behaviour that consumers are interested in.